blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/BitbucketApi.java

```diff
@@ -27,26 +24,19 @@
  */
 @Restricted(NoExternalUse.class)
 public abstract class BitbucketApi {
+
     protected final String apiUrl;
-    protected final StandardUsernamePasswordCredentials credentials;
     protected final String userName;
-    protected final String basicAuthHeaderValue;
+    protected final HttpRequest request;
 
     //XXX: To be used for testing to resolve correct factory
     public static final String X_BB_API_TEST_MODE_HEADER="X_BB_API_TEST_MODE_HEADER";
```

Comment:
is this meant to be left in here?

Reply:
Ah, this is old code put there early on to facilitate wiremock test. I am going to get rid of it.